styles/index.css

```diff
@@ -1,6 +1,15 @@
 @tailwind base;
 /* Write your own custom base styles here */
 
+input[type='search']::-webkit-search-cancel-button {
```

Comment:
does this need to be here because of the `::-webkit-search-cancel-button ` ?

Reply:
oh sorry yea bad wording - was curious why this couldn't be done with a Tailwind class of some sort, but I think the answer is that Tailwind doesn't support the ::-webkit-search-cancel-button pseudo element